commons/pac-batch-commons/src/main/java/com/tmobile/pacman/commons/dto/PermissionVH.java

```diff
@@ -1,15 +1,19 @@
 package com.tmobile.pacman.commons.dto;
 
+import java.util.List;
+import java.util.Map;
+
 public class PermissionVH {
-    ErrorVH errorVH;
+    Map<String, List<String>> assetPermissionIssue;
```

Comment:
This should be plural; assetPermissionIssues

Reply:
Addressed in commit